spec.emu

```diff
@@ -1330,6 +1347,74 @@ location: https://github.com/tc39/proposal-decimal/
       </emu-clause>
     </emu-clause>
   </emu-clause>
+
+  <emu-clause id="sec-withfractionaldigits">
+    <h1>Numerics with Fractional Digits</h1>
+    <p>A WithFractionalDigits is an object that specifies how another object, such as a Decimal128, can be enriched with additional information about its precision and how that additional precision can be presented. WithPrecision objects are not instantiable; they are created, instead, by calling a suitable method in any object that has the `withPrecision` method in its prototype.</p>
+    <emu-clause id="sec-withfractionaldigits-prototype">
+      <h1>The %WithFractionalDigitsPrototype% Object</h1>
+      <p>The <dfn>%WithFractionalDigitsPrototype%</dfn> object:</p>
+      <ul>
+        <li>has properties that are inherited by all WithFractionalDigits Objects.</li>
+        <li>is an ordinary object.</li>
+        <li>has a [[Prototype]] internal slot whose value is %WithFractionalDigitsPrototype%.</li>
+        <li>has the following properties:</li>
+      </ul>
+
+      <emu-clause id="sec-%withprecisionprototype%.number">
+        <h1>get %WithFractionalDigitsPrototype%.number</h1>
+        <p>The underlying <q>number</q> value. It may not necessarily be a Number.</p>
+      </emu-clause>
+
+      <emu-clause id="sec-%withprecisionprototype%.precision">
+        <h1>get %WithFractionalDigitsPrototype%.precision</h1>
+        <p>The <q>precision</q> that the number is understood to have.</p>
```

Comment:
Agree. I'm not happy with the names, either. I chose "precision" because that's what we use instead of "quantum". I'm happy to just roll with these names and bikeshed later.